http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -143,9 +143,9 @@ public enum RelativeUrlsUsed {
     protected String requestObjectSigningKeyAlias;
     protected String requestObjectSigningKeyStoreType;
     protected JWKEncPublicKeyLocator encryptionPublicKeyLocator;
-    private boolean logoutSessionRequired = true;
+    private String logoutSessionRequired;
```

Comment:
Why was this changed to a String?

Reply:
This is an optional setting.  We needed to be able to determine if the user specifically set the value or not.
If this was a boolean we would not be able to determine if the user set the value to true, false or not set at all.

Reply:
From the analysis document, it looks like the default for this should be set to true. We should be using `boolean` here (we have other similar boolean attributes as well). (Note that it's also possible to use `Boolean` but I think that will be needed for the system property case, it shouldn't be needed here.)

Reply:
done